lib/gui/index.html

```diff
@@ -32,18 +32,18 @@
 
     <footer class="section-footer" ng-controller="StateController as state"
       ng-hide="state.is('success')">
-      <svg-icon path="../../../assets/etcher.svg"
-        width="83px"
-        height="13px"
+      <svg-icon path="'../../assets/etcher.svg'"
+        width="'83px'"
+        height="'13px'"
         os-open-external="https://etcher.io?ref=etcher_footer"></svg-icon>
 
       <span class="caption">
         is <span class="caption" os-open-external="https://github.com/resin-io/etcher">an open source project</span> by
       </span>
 
-      <svg-icon path="../../../assets/resin.svg"
-        width="79px"
-        height="23px"
+      <svg-icon path="'../../assets/resin.svg'"
+        width="'79px'"
+        height="'23px'"
```

Comment:
Yeah, React basically treats data in attributes like regular Javascript, so to get strings you have to encapsulate them in quotes, and everything else you'd normally do. Looks odd because it's in quotes already, but this is temporary while we still have Angular in our codebase—once we're pure React we can use JSX and the syntax is a bit cleaner:

```jsx
<Svg-Icon path={ '../../assets/resin.svg' } ...></Svg-Icon>
```